packages/selected-network-controller/tests/SelectedNetworkMiddleware.test.ts

```diff
@@ -58,7 +58,7 @@ describe('createSelectedNetworkMiddleware', () => {
     expect(req.networkClientId).toBe('mockNetworkClientId');
   });
 
-  it('sets the networkClientId for the domain to the current network from networkController if one is not set', async () => {
+  it('puts metamask selected networkClientId on request if requesting origin does not have one set', async () => {
```

Comment:
```suggestion
  it('puts metamask selected networkClientId on the request object if the requesting origin does not have one set', async () => {
```

Reply:
or just `on the request` works too

Reply:
done here: https://github.com/MetaMask/core/pull/3908/commits/ecb5e72a77608459747c279a2e14aad4c94b63a1